ci/envs/310-dev.yaml

```
@@ -26,7 +26,7 @@ dependencies:
     - geopy
     - mapclassify>=2.4.0
     # dev versions of packages
-    - --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+    - --upgrade --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
     - --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple pandas
```

I did some more testing on this, and I think numpy is just getting parsed out from the end of this line
```suggestion
    - --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple
    - numpy
    - pandas
```

---

I've commited this suggestion to see if it works on ci as well, hope you don't mind.

---

Certainly not!

---

And it seems to have worked, thanks for figuring it out!